dump.go

```diff
@@ -34,6 +35,7 @@ type Options struct {
 var Config = Options{
 	StripPackageNames: false,
 	HidePrivateFields: true,
+	FieldExclusions:   regexp.MustCompile(`^(XXX_.*)$`),
```

Comment:
Doing this by default is probably uncontroversial, but it should have a comment describing what `XXX_` is.

Reply:
Indeed, comment added